src/helpers/_font-faces.scss

```diff
@@ -7,60 +7,34 @@
 
 @import "../tools/exports";
 
-/// Font Face - NTA
+/// Font Face - GOV.UK Transport
 ///
-/// Outputs the font-face declaration for NTA at the root of the CSS document
+/// Outputs the font-face declaration for GOV.UK Transport at the root of the CSS document
 /// the first time it is called.
 ///
 /// @access private
 
-@mixin _govuk-font-face-nta {
-  @include govuk-exports("govuk/helpers/font-face") {
+@mixin _govuk-font-face-govuk-transport {
+  @include govuk-exports("govuk/helpers/font-faces") {
     @at-root {
       /*! Copyright (c) 2011 by Margaret Calvert & Henrik Kubel. All rights reserved. The font has been customised for exclusive use on gov.uk. This cut is not commercially available. */
 
       @font-face {
-        font-family: "nta";
-        src: govuk-font-url("light-2c037cf7e1-v1.eot");
-        src: govuk-font-url("light-2c037cf7e1-v1.eot?#iefix") format("embedded-opentype"),
-             govuk-font-url("light-f38ad40456-v1.woff2") format("woff2"),
-             govuk-font-url("light-458f8ea81c-v1.woff") format("woff");
+        font-family: "GOVUKTransport";
+        src: govuk-font-url("light-68f0c84f0e-v2.woff2") format("woff2"),
+             govuk-font-url("light-222368e53d-v2.woff") format("woff");
         font-weight: normal;
         font-style: normal;
-        font-display: fallback;
+        font-display: swap;
```

Comment:
@hannalaakso has noticed the behaviour might not be better than fallback.

We should leave this as fallback, and do a proper test that includes slower network conditions and an optimised server before changing this.

Reply:
https://github.com/alphagov/govuk-frontend/issues/1437